components/multi-platform-controller/production-downstream/host-config.yaml

```diff
@@ -60,7 +60,7 @@ data:
   dynamic.linux-arm64.aws-secret: aws-account
   dynamic.linux-arm64.ssh-secret: aws-ssh-key
   dynamic.linux-arm64.security-group-id: sg-0903aedd465be979e
-  dynamic.linux-arm64.max-instances: "50"
+  dynamic.linux-arm64.max-instances: "250"
```

Comment:
We cannot increase that before we get a new CIDR block and increase the number of IPs on AWS side, we only have 128 ips for now. The actual config is already broken, if we sum up the number of VMs that can be spawned concurrently, it is already more than 128

Reply:
I'm going to drop the PR then because I opened it based on bad info